README.md

```diff
@@ -12,53 +12,72 @@ For running Osmose Frontend with Docker go into the docker directory. You can al
 Manual Installation
 -------------------
 
+### Dependencies
+
+For Ubuntu Server 16.04 and 18.04
+
+```
+apt install python2.7 python2.7-dev virtualenv gcc pkg-config libpng-dev libjpeg-dev libfreetype6-dev postgresql-server-dev-all libgeos-dev g++ python-shapely nodejs npm
+```
+
+#### Database
+
+```
+apt install postgresql postgresql-contrib
+```
+
+#### Web server
+
+```
+apt install apache2 libapache2-mod-wsgi
+```
+
+#### Internationalisation and SVG rendering
+
+```
+apt install gettext librsvg2-bin
+```
+
 ### Initialisation
 
 Generate translation files
 ```
-cd po && make mo
+cd web/po && make mo
 ```
 
 
 ### Python
 
 Osmose QA frontend requires python > 2.6 and < 3
 
-Setup system dependencies (Ubuntu Server 16.04)
-```
-apt install python2.7 python2.7-dev virtualenv gcc pkg-config libpng-dev libjpeg-dev libfreetype6-dev postgresql-server-dev-all libgeos-dev g++ python-shapely nodejs
-```
-
 Create a python virtualenv, active it and install python dependencies
 ```
 virtualenv --python=python2.7 osmose-frontend-venv
 source osmose-frontend-venv/bin/activate
-pip install -r requirements.txt
 ```
 
-Generate markers
+Workaround for Ubuntu 16.04, as pip install the [latest setuptools](https://github.com/tilezen/mapbox-vector-tile/blob/e16fca20ec4e45e12caff8393150ff3327ad1a4d/setup.py#L33),
```

Comment:
We are in 2020, I think we can drop 16.04 support.

Reply:
Ok I will drop 16.04 support from the README.